integration_tests/src/main/python/get_json_test.py

```diff
@@ -37,8 +37,7 @@ def test_get_json_object(json_str_pattern):
             'get_json_object(a, "$.store.fruit[0]")',
             'get_json_object(\'%s\', "$.store.fruit[0]")' % scalar_json,
             ),
-        conf={'spark.sql.parser.escapedStringLiterals': 'true',
-            'spark.rapids.sql.expression.GetJsonObject': 'true'})
+        conf={'spark.sql.parser.escapedStringLiterals': 'true'})
```

Comment:
Now `GetJsonObject` is on by default. Do we have this option configured somewhere so we remove it too?

Reply:
Yes the same option is still there, users can disable it if they want. But we don’t need to set it on in tests because it is on by default now.